engine/testutil/nodes.go

```diff
@@ -594,7 +594,15 @@ func ExecutionNode(t *testing.T, hub *stub.Hub, identity *flow.Identity, identit
 		fvm.WithInitialTokenSupply(unittest.GenesisTokenSupply))
 	require.NoError(t, err)
 
-	err = bootstrapper.BootstrapExecutionDatabase(node.PublicDB, commit, genesisHead)
+	rootResult := unittest.ExecutionResultFixture()
+	unittest.WithExecutionResultBlockID(genesisHead.ID())(rootResult)
+	unittest.WithFinalState(commit)(rootResult)
+
+	// TODO: use state commitment from BootstrapLedger?
+	rootSeal := unittest.Seal.Fixture()
+	unittest.Seal.WithResult(rootResult)(rootSeal)
```

Comment:
I agree with this TODO - it seems wrong to use a fixture here. Can we read the real root seal from `node.State` instead?